### PR TITLE
fix(frontend): solve console warning

### LINKED ISSUE #609: Fix console warning
When loading Lifebank, console show up error messages.

image

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #609 

### What does this PR do?
Solve console warnings

### How should this be tested?
- Make run
- Check there is no warning

### CODE REVIEW

**Comment by leisterfrancisco** [comment]
Please @tetogomez @JeanVegaD take a look a some of these changes, I'm make the console solving recommendation. Most of the error are on the `useEffect`.

**Review by tetogomez — APPROVED** [review]
Good job @leisterfrancisco

**Review by steph222 — APPROVED** [review]

**Review by JustinCast — APPROVED** [review]

File: webapp/src/routes/Profile/ProfilePageSponsor.js [hunk 1]
```diff
@@ -621,7 +614,7 @@ const ProfilePageSponsor = ({ profile }) => {
           <Box className={classes.rowBoxLeft}>
             <Typography className={classes.rowTitle} variant="subtitle1">{t('profile.images')}</Typography>
             <Box>
-              <img className={classes.img} src={images[activeStep]} />
+              <img className={classes.img} src={images[activeStep]} alt='Profile sponsor' />
```
Comment (JustinCast) [inline webapp/src/routes/Profile/ProfilePageSponsor.js:617]: Maybe we can use alt in english and spanish

File: webapp/src/routes/Profile/ProfilePageLifebank.js [hunk 2]
```diff
@@ -517,7 +517,7 @@ const ProfilePageLifebank = ({ profile }) => {
           <Box className={classes.rowBoxLeft}>
             <Typography className={classes.rowTitle} variant="subtitle1">{t('profile.images')}</Typography>
             <Box>
-              <img className={classes.img} src={images[activeStep]} />
+              <img className={classes.img} src={images[activeStep]} alt='Lifebank profile' />
```
Comment (JustinCast) [inline webapp/src/routes/Profile/ProfilePageLifebank.js:520]: We can use **alt** in both languages

File: webapp/src/routes/InfoPage/InfoPageDesktop.js [hunk 3]
```diff
@@ -335,6 +288,53 @@ const InfoPage = () => {
...
+      }
+    }
+  }, [getInfo, getOffers, profile, location])
+
+  useEffect(() => {
+    if (errorUsername && errorInfoProfile) {
+      if (errorUsername.message === 'GraphQL error: Could not verify JWT: JWTExpired') {
+        logout()
+        history.push(`/info/${location.state.profile.account}`)
+      } else history.push('/internal-error')
```
Comment (JustinCast) [inline webapp/src/routes/InfoPage/InfoPageDesktop.js:321]: Sometimes you use no curly braces when there is just one single statement in the body of a comparison, sometimes not. Maybe we can be consistent and use either one or the other

Comment (leisterfrancisco) [reply webapp/src/routes/InfoPage/InfoPageDesktop.js:321]: @JustinCast I didn't make that change, I don't really know why is marked as I edited that but It wasn't

File: webapp/src/routes/Profile/ProfilePageLifebank.js (hunk 2 above)
Comment (leisterfrancisco) [reply webapp/src/routes/Profile/ProfilePageLifebank.js:520]: You right, working on

File: webapp/src/routes/Profile/ProfilePageSponsor.js (hunk 1 above)
Comment (leisterfrancisco) [reply webapp/src/routes/Profile/ProfilePageSponsor.js:617]: You right, working on

File: webapp/src/routes/InfoPage/InfoPageMobile.js [hunk 4]
```diff
@@ -340,13 +325,27 @@ const InfoPageMobile = () => {
...
 
+  useEffect(() => {
+    getInfo()
+  }, [getInfo, location])
+
+  const getOffers = async () => {
+    if(profile){
+      if(profile.role === 'sponsor'){
+      setLoadingOffers(true)
+      await getAllOffers()
```
Comment (JustinCast) [inline webapp/src/routes/InfoPage/InfoPageMobile.js:336]: Check the indentation

Comment (leisterfrancisco) [reply webapp/src/routes/InfoPage/InfoPageMobile.js:336]: @JustinCast just solved, just to remember, the purpose of this PR is to solve the console warnings, not indentations or similar errors coming from others merge changes.

Comment (JustinCast) [reply webapp/src/routes/InfoPage/InfoPageMobile.js]: Ok, got it. Good job

**Comment by JustinCast** [comment]
@leisterfrancisco take a look at the conflict

**Comment by leisterfrancisco** [comment]
> @leisterfrancisco take a look at the conflict

@JustinCast thanks, working on

**Review by JustinCast — APPROVED** [review]

**Review by Edgar097 — APPROVED** [review]

**Pull request merged**